python/cudf/cudf/tests/test_json.py

```diff
@@ -936,14 +936,35 @@ def test_json_dtypes_nested_data():
         """,
         ),
     ],
+    # TODO failing test cases
```

Comment:
Is there still a TODO here, or can this line be removed?

Reply:
added a failing test case.